### ISSUE TITLE
Add option to preserve artifact in the build folder

### ISSUE THREAD

**Issue by mlarcher** [micha149/gulp-maven-deploy#12]
It would be nice to be able to keep the generated artifact in the build folder, for archiving by the CI. Any chance this could be done ?

**Comment by leftieFriele** [comment]
I haven't been working in a Java environment the past year so I haven't given much love to this project. The fastest thing is probably to dig in and submit a pr, sorry.

**Comment by micha149** [comment]
For a better gulp feeling, I planned to remove the packaging entirely from this plugin and concentrate on the job „deploy“. In this case you could pipe a packaged file to the deploy plugin and after that to `gulp.dest()`. Here is a quick example:

```javascript
gulp.src('src/*')
    .pipe(zip('myArtifact.war'))
    .pipe(deploy(mavenDeployOptions))
    .pipe(gulp.dest('dist/'));
```

Would this work for you?

**Comment by mlarcher** [comment]
sounds good to me

**Comment by micha149** [comment]
Great to hear. So, I am on it…

**Comment by micha149** [comment]
Yesterday I released version `1.0.0-beta.5` wich brings the announced behavior. Can you check if this works as expected by you? I it does, I would like to release version `1.0.0` in two weeks.
